eng/helix/payload/installazurite.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
```

Comment:
I think the main exectuable.sh script goes through an marks everything executable, but I've updated it so that they are executable by default. For some reason, I thought I already did this, but apparently not.